app/src/main/res/menu/bottom_nav_menu.xml

```diff
@@ -3,17 +3,17 @@
 
     <item
         android:id="@+id/navigation_dashboard"
-        android:icon="@drawable/ic_dashboard_black_24dp"
+        android:icon="@mipmap/walletbalance_foreground"
```

Comment:
These icons look great, but I think we should try to make them a bit bigger. Especially the reddit logo looks pretty small